### PR TITLE
Feature/image format conversion

### PR DESCRIPTION
**Pull request by tofixx**

### CODE REVIEW

File: public/js/test.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ $('#compress').slider({
 function updateImageUrl() {
   var values = $('#configure-form').serialize();
   $('#result-image').attr('src', '/image?' + values);
+  $('#result-image-thumbnail').css('width', $('#width').val() + 'px');
+  $('#result-image-thumbnail').css('height', $('#height').val() + 'px');
```
Comment (franticBIT) [inline public/js/test.js]: Could you make this fit the page rather than show it at full scale?

File: views/test.jade [hunk 2]
```diff
@@ -60,31 +60,33 @@ block content
...
 
                     div.form-group
-                        div.col-sm-offset-2.col-sm-6
+                        div.col-sm-offset-4.col-sm-8
                             button#update-button.btn.btn-primary(type='submit')= 'do the magic...'
-            div.col-sm-4
-                img#result-image.thumbnail(src='https://placeholdit.imgix.net/~text?txtsize=33&txt=350%C3%97150&w=350&h=150')
+            div.col-sm-6
+                div#result-image-thumbnail.image
+                    img#result-image(src='https://placeholdit.imgix.net/~text?txtsize=33&txt=350%C3%97150&w=350&h=150')
```
Comment (franticBIT) [inline views/test.jade]: What about a download button? 😁

File: public/js/test.js (hunk 1 above)
Comment (tofixx) [inline public/js/test.js]: maybe later.

File: views/test.jade (hunk 2 above)
Comment (tofixx) [inline views/test.jade]: added direct link to image, without download

**Pull request merged**